packages/master-detail-layout/src/vaadin-master-detail-layout.js

```diff
@@ -417,6 +421,26 @@ class MasterDetailLayout extends SlotStylesMixin(ResizeMixin(ElementMixin(Themab
     }
   }
 
+  /** @private */
+  __onBackdropClick() {
+    this.dispatchEvent(
+      new CustomEvent('backdrop-click', {
+        bubbles: true,
```

Comment:
Should the events propagate through the DOM? Wouldn't that complicate things with nested layouts? Theoretically there could also be conflicts with the somewhat generic event names.

Reply:
Good point, I missed that. Removed `bubbles` from both events.